django_webapps/scripts/db_population/populate_coordinators.py

```diff
@@ -0,0 +1,106 @@
+import datetime
+import os
+from dataclasses import dataclass
+
+import django
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "django_webapps.settings")
+django.setup()
+
+from django.db import IntegrityError, transaction  # noqa: E402
+from django.db.transaction import TransactionManagementError  # noqa: E402
+from django.utils import timezone  # noqa: E402
+
+from api.models.choices.status.project import CoordinatorProjectStatus  # noqa: E402
+from api.models.coordinator import Coordinator  # noqa: E402
+from django_webapps.scripts.db_population.base_populator import (  # noqa: E402
+    BasePersonCsvPopulator,
+    BasePersonEntityData,
+    CsvData,
+)
+from django_webapps.scripts.db_population.parsers import common_parsers  # noqa: E402
+from django_webapps.scripts.db_population.utils import (  # noqa: E402
+    get_args,
+    get_logger,
+    load_csv_data,
+)
+
+logger = get_logger("coordinators.log")
+LIST_OF_ADMINS_IDS = [22, 42]
+MAX_TID_WITH_NO_RESPONSE = 1300
+
+
+COLUMN_TO_ID = {
+    "tid": 0,
+    "name": 1,
+    "email": 2,
+    "tg": 3,
+    "timezone": 4,
+}
+
+
+@dataclass
+class CoordinatorData(BasePersonEntityData):
+    project_status: CoordinatorProjectStatus = CoordinatorProjectStatus.WORKING_OK
```

Comment:
Maybe introduce some Legacy status?

Reply:
There is not such status

Reply:
No, I mean that if there are some legacy statuses in Google Sheets, we can put a comment saying `"Status before conversion": [some status from Google Sheet]`

Reply:
Yes, I meant there is no any status in the google sheet

Reply:
oh, sorry. So by "introducing legacy status" you meant that they should go to Google Sheets and add something there? OK, got it. Well, let's just put `WORKING_OK` for now. After that, during conversion of groups, we should be able to change them to more precise ones where necessary.